### PR TITLE
Bugfix/leaf 1787 fix update database scripts

### PR DESCRIPTION
**Pull request by jeremyrl**

### CODE REVIEW

File: LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+START TRANSACTION;
+
+CREATE TABLE `test` (
```
Comment (patrickcgs) [inline LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql:3]: ?

Comment (dastan-rahimi) [reply LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql:3]: why do we need this db script?

Comment (dastan-rahimi) [reply LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql:3]: This DB script is no longer needed please remove.

Comment (jeremyrl) [reply LEAF_Nexus/db_upgrade/Update_OC_DB_202062600-2020082400.sql:3]: done

File: LEAF_Request_Portal/scripts/updateDatabase.php [hunk 2]
```diff
@@ -2,7 +2,9 @@
 /*
  * As a work of the United States government, this project is in the public domain within the United States.
  */
-
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
```
Comment (thomasnguyen95) [inline LEAF_Request_Portal/scripts/updateDatabase.php:6]: do we still need all these error logs? 
If yes, why not for the ```LEAF_Nexus/scripts/updateDatabase.php```

**Pull request merged**